Makefile

```diff
@@ -90,8 +90,11 @@ INV_DIR := $(shell pwd)
 ifeq (cygwin,$(OSTYPE))
 #JAVA tools need Windows path on Windows
 JAR_DIR := $(shell cygpath -m $(INV_DIR))
+# for install-test target
+QT_PATH:=`cygpath -wp ../../../daikon.jar:.`
```

Comment:
It's inconsistent that line 92 uses $(shell ...) and this line uses `...`.  I would make the two consistent.